Case Management - Generic - Updates from Customer Success (#20149)
demisto/content: Packs/CaseManagement-Generic/ReleaseNotes/1_3_0.md

##### New: Case Management - Generic - Set SLAs based on Severity
- Added the **Case Management - Generic - Set SLAs based on Severity** playbook, sets the SLAs for Incidents, the Time to Assignment Timer, and the Remediation SLA Timer based on the Incident Severity.

#### Scripts
##### GenerateSummaryReportButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### New: CaseMgmtAnalystTools
- Added the **CaseMgmtAnalystTools**, dynamic display script to display a list of useful Analyst Tools on an Incident layout. (Available from Cortex XSOAR 6.2.0).

##### New: CaseMgmtIncidentTypesDisplay
- Added the **CaseMgmtIncidentTypesDisplay**, field display script that restricts the Incident Types a user can create manually based on an XSOAR list, and prevents changing the Incident Type manually once it is created. (Available from Cortex XSOAR 6.2.0).

##### LinkIncidentsButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### AssignToMeButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### TimersOnOwnerChange
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.

#### Widgets
##### My Incidents by Type

> - Added the **My Incidents by Type**.
> 
> ##### Mean Time to Remediation (Remediation SLA)
> - Added the **Mean Time to Remediation (Remediation SLA)**.
> 
> ##### Mean Time to Assignment (Time to Assignment)
> - Added the **Mean Time to Assignment (Time to Assignment)**.
> 
> ##### Participating Incidents
> - Added the **Participating Incidents**..
> 
> ##### My Mean Time to Remediation (Remediation SLA)
> - Added the **My Mean Time to Remediation (Remediation SLA)**.
> 
> ##### Participating Incidents Count
> - Added the **Participating Incidents Count**.
> 
> #### Layouts
> ##### Case layout
> - Added the **My Incidents by Type**.
> 
> #### Reports
> ##### Case Report
> - Added the **Case Report**.

Reviewer: ```suggestion
- Improved implementation to the **My Incidents by Type** widget.

##### Mean Time to Remediation (Remediation SLA)
- Improved implementation to the **Mean Time to Remediation (Remediation SLA)** widget.

##### Mean Time to Assignment (Time to Assignment)
- Improved implementation to the **Mean Time to Assignment (Time to Assignment)** widget.

##### Participating Incidents
- Improved implementation to the **Participating Incidents** widget.

##### My Mean Time to Remediation (Remediation SLA)
- Improved implementation to the **My Mean Time to Remediation (Remediation SLA)** widget.

##### Participating Incidents Count
- Improved implementation to the **Participating Incidents Count** widget.

#### Layouts
##### Case layout
- Improved implementation to the **Case layout**.

#### Reports
##### Case Report
- Improved implementation to the **Case Report**.
```

##### New: Case Management - Generic - Set SLAs based on Severity
- Added the **Case Management - Generic - Set SLAs based on Severity** playbook, sets the SLAs for Incidents, the Time to Assignment Timer, and the Remediation SLA Timer based on the Incident Severity.

#### Scripts
##### GenerateSummaryReportButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### New: CaseMgmtAnalystTools
- Added the **CaseMgmtAnalystTools**, dynamic display script to display a list of useful Analyst Tools on an Incident layout. (Available from Cortex XSOAR 6.2.0).

##### New: CaseMgmtIncidentTypesDisplay
- Added the **CaseMgmtIncidentTypesDisplay**, field display script that restricts the Incident Types a user can create manually based on an XSOAR list, and prevents changing the Incident Type manually once it is created. (Available from Cortex XSOAR 6.2.0).

##### LinkIncidentsButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### AssignToMeButton
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
##### TimersOnOwnerChange
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.

> <!--

Reviewer: ```suggestion

```

#### Widgets
##### My Incidents by Type
- Added the **My Incidents by Type**.

##### Mean Time to Remediation (Remediation SLA)
- Added the **Mean Time to Remediation (Remediation SLA)**.

##### Mean Time to Assignment (Time to Assignment)
- Added the **Mean Time to Assignment (Time to Assignment)**.

##### Participating Incidents
- Added the **Participating Incidents**..

##### My Mean Time to Remediation (Remediation SLA)
- Added the **My Mean Time to Remediation (Remediation SLA)**.

##### Participating Incidents Count
- Added the **Participating Incidents Count**.

#### Layouts
##### Case layout
- Added the **My Incidents by Type**.

#### Reports
##### Case Report
- Added the **Case Report**.

> -->

Reviewer: ```suggestion

```
